go/internal/airtable_fetcher/client.go

```diff
@@ -5,23 +5,31 @@ type Client struct {
 }
 
 type ClientConfig struct {
-	APIKey           string
-	BaseID           string
-	LaunchpadTableID string
-	BannersTableID   string
-	BannersViewID    string
-	NewsTableID      string
-	NewsViewID       string
+	APIKey            string
+	APIKeydappsStore  string
+	BaseID            string
+	DAppStoreBaseID   string
+	LaunchpadTableID  string
+	BannersTableID    string
+	BannersViewID     string
+	NewsTableID       string
+	NewsViewID        string
+	dAppGroupsTableID string
+	dAppTableID       string
 }
 
 func NewClient(apiKey string) *Client {
 	return &Client{config: ClientConfig{
-		APIKey:           apiKey,
-		BaseID:           "appetXQzVoElrsJs5",
-		LaunchpadTableID: "tbla2ZD8MtljtvHKt",
-		BannersTableID:   "tblUXvHaNCABlq8zg",
-		BannersViewID:    "viwHEB0KsHyF7EuUQ",
-		NewsTableID:      "tblOlJKlbNsSpdAVq",
-		NewsViewID:       "viwzEqf2inBThqoFY",
+		APIKey:            apiKey,
+		APIKeydappsStore:  apiKey,
```

Comment:
I see that we have 2 separated configs: APIKey and APIKeyDappStore from .env, why here we use the same key ? if it's the same can we use only one key ?